src/App.vue

```diff
@@ -138,7 +138,8 @@
             </v-col>
 
             <v-col cols="12" lg="3">
-              <aside>
+              <!-- Render fee summary only after data is loaded. -->
+              <aside v-if="haveData">
```

Comment:
Still displays (at same time as spinner is hidden):

![image](https://user-images.githubusercontent.com/10002889/215149132-c4c77b64-12f1-4fc9-9852-f17f7b58484f.png)